xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/CachedModelBridge.java

```diff
@@ -95,6 +96,13 @@ public void setStartDateForUser(DocumentReference userReference, Date startDate)
     }
 
     @Override
+    public PageNotificationEventDescriptor getPageNotificationEventDescriptor(DocumentReference documentReference)
+            throws NotificationException
+    {
+        return modelBridge.getPageNotificationEventDescriptor(documentReference);
```

Comment:
Ok so add this explanation as a comment in the code, so nobody else will ask the question :)